### DISCUSSION TITLE
How can I use the latest version of SDL2?

### DISCUSSION THREAD

**Discussion by km19809** [emscripten-core/emscripten#17652] [Q&A] [answered]
When I trying to link SDL, Emscripten fetches:
`ports:INFO: retrieving port: sdl2 from https://github.com/libsdl-org/SDL/archive/release-2.0.20.zip`
However, the latest version is [2.0.22](https://github.com/libsdl-org/SDL/releases/tag/release-2.0.22).

Actually, what I want is to use the pre-release version, but I could not find out how to use it instead of 2.0.20.
Can I modify something or wait until the emscripten update?

**Comment by kripken** [comment, marked as answer]
You can modify `tools/ports/sdl2.py` to tell it to use a different hash/commit. See `TAG` and `HASH` near the top.
